portal-test-integration/src/com/liferay/portal/test/rule/AspectJNewEnvTestRule.java

```diff
@@ -158,8 +163,20 @@ public SwitchClassLoaderProcessCallable(
 
 			_dumpDir = dumpDir;
 
-			_encodedProcessCallable = SerializableUtil.serialize(
-				processCallable);
+			UnsyncByteArrayOutputStream unsyncByteArrayOutputStream =
```

Comment:
@dantewang @ChiZelin 
Please use Serializer and Deserializer here. Take a look at the corresponding unit test to check how to use it.

Tina.